### ISSUE TITLE
Sync Mode Light

### ISSUE THREAD

**Issue by cryptojunky** [ethereum/mist#3104]
```
Version: `0.9.1`
OS & Version:  Windows X
Node version: `geth latest` 
Number of blocks synchronized: 4,322,878
```
Once the block count finishes from the beginning (which is a fast download) it bounces back and forth between 3 minutes since last block and 2 minutes since last block and after quite a bit of time it starts counting blocks as it should. Than it will start back to 2 or 3 minutes since last block and stay there.

**Comment by ericjarvies** [comment]
There is/are problem/s with 0.9.1 (Ethereum/Mist), wherein it is not importing new chain segments correctly/properly.

If you try to rollback/use the 0.9.0 app(s) against the same [_Ethereum/Mist_] data folders, the problem will remain.  Unfortunately, you will need to delete **all** of the exiting [**Ethereum/Mist]** app folders (e.g.- Library/Ethereum/* etc.) and start over (_with 0.9.0 or earlier_).

**Comment by evertonfraga** [comment]
No, please don't delete all Library/Ethereum files. All accounts visible by
geth would vanish as well.

Mist and EW share the same `geth` binary instance. The problem you're
facing has to to with Geth, not Mist.

Please refer to this small guide, and post your results:
https://github.com/ethereum/mist/wiki#start-the-node-manually



--
Everton Fraga

**Comment by cryptojunky** [comment]
It says: warn blockchain not empty, fast sync disabled.

When in the wallet it continues downloading normal blocks and disables the new [Sync with Light client (beta)]

I am only a few days behind on the blockchain so the wallet will catch up if need be. 

I still have no problem running the light version through the shortcuts explained here: 
https://www.reddit.com/r/ethereum/comments/669cn9/how_to_use_the_geth_light_client_in_combination/
